src/app/contacts-app/contacts-app.component.ts

```diff
@@ -82,6 +83,8 @@ export class ContactsAppComponent {
             const target = this.groupFilter.substr(5); // strip 'USER:'
 
             return c.categories.find(g => g === target);
+        }).filter(c => {
+            return c.display_name().toLowerCase().indexOf(this.searchTerm.toLowerCase()) != -1;
```

Comment:
It is indeed possible that it returns `null` sometimes, [and we even take advantage of that when displaying them](https://github.com/runbox/runbox7/pull/158/commits/a2ce5f20cb6b01394b2d14945dcd1ec342fa91c7).

I'll check for it here to stop it from crashing, and alter the `display_name()` signature so that it reflects it.

Reply:
Addressed in 1634e1b